index.html

```diff
@@ -8,27 +8,232 @@
+    </div>
+    <div class="content">
+        <h3>Project name goes here</h3>
+        <p>
+            <ul>
+                <li class="item1">HTML/CSS</li>
+                <li class="item2">Ruby Rails</li>
+                <li class="item3">JavaScript</li>
+            </ul>
+        </p>
+        <button>See this project <img src="images/arrow.png"> </button>
+    </div>
+</div>
+</div>
+</div>
+
+<!-- 4th project -->
+<div class="card grid-item">
+    <div class="row">
+        <div class="column">
+    <div class="photo">
+        <img src="images/proj4.png" alt="Shopping_Website">
+    </div>
+    <div class="content">
+        <h3>Project name goes here</h3>
+        <p>
+            <ul>
+                <li class="item1">HTML/CSS</li>
+                <li class="item2">Ruby Rails</li>
+                <li class="item3">JavaScript</li>
+            </ul>
+        </p>
+        <button>See this project <img src="images/arrow.png"> </button>
+    </div>
+</div>
 </div>
+</div>
+
+<!-- 5th project -->
+<div class="card grid-item">
+    <div class="row">
+        <div class="column">
+    <div class="photo">
+        <img src="images/proj5.png" alt="Shopping_Website">
+    </div>
+    <div class="content">
+        <h3>Project name goes here</h3>
+        <p>
+            <ul>
+                <li class="item1">HTML/CSS</li>
+                <li class="item2">Ruby Rails</li>
+                <li class="item3">JavaScript</li>
+            </ul>
+        </p>
+        <button>See this project <img src="images/arrow.png"> </button>
+    </div>
+</div>
+</div>
+</div>
+
+<!-- 6th project -->
+<div class="card grid-item">
+    <div class="row">
+        <div class="column">
+    <div class="photo">
+        <img src="images/proj6.png" alt="Shopping_Website">
+    </div>
+    <div class="content">
+        <h3>Project name goes here</h3>
+        <p>
+            <ul>
+                <li class="item1">HTML/CSS</li>
+                <li class="item2">Ruby Rails</li>
+                <li class="item3">JavaScript</li>
+            </ul>
+        </p>
+        <button>See this project <img src="images/arrow.png"> </button>
+    </div>
+</div>
+</div>
+</div>
+</section>
+<!-- About US section -->
+<section>
+<div class="card grid-item about-setngs">
+    <div class="header about">
+        <h2>About</h2>
+    </div>
+    <div class="row">
+        <div class="column">
+    <div class=" about_para">
+        <p class="about_bg">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        
+        <button class="dwnld_btn">Get my CV <img src="images/download.png"> </button>
+    </div>
+</div>
+  </div>
+</div>
+
+<!-- About_us card-1 -->
+<div class="bg-info">
+    <div class="info">
+        <div class="skills">
+        <h1>Languages</h1>
```

Comment:
There should be only one `<h1>` tag on a page for semantic and SEO reasons. Kindly replace `<h1>` tag other than used for your name in the first section with `hx` tags (h2. .... h6).